lib/puma/const.rb

```diff
@@ -120,21 +120,6 @@ module Const
 
     PUMA_TMP_BASE = "puma"
 
-    ERROR_RESPONSE = {
-      # Indicate that we couldn't parse the request
-      400 => "HTTP/1.1 400 Bad Request\r\n\r\n",
-      # The standard empty 404 response for bad requests.  Use Error4040Handler for custom stuff.
-      404 => "HTTP/1.1 404 Not Found\r\nConnection: close\r\nServer: Puma #{PUMA_VERSION}\r\n\r\nNOT FOUND",
```

Comment:
This change means we no longer have `NOT FOUND` in a 404 and `BUSY` in a 503 response, right?

Reply:
The original constant has defined 404 and 503 keys, but I don't think Puma used the string values anywhere.

In the first post, I accidentally left out two responses generated in `Request`, a 500 and a 503, which are used in calls to 
`Server#lowlevel_error`.

I think these responses should not be 'specific' to Puma, they should be generic, so one can't tell how they were generated?

But, [9110 15.6. Server Error 5xx](https://www.rfc-editor.org/rfc/rfc9110.html#name-server-error-5xx) states the below, which kind of implies a body...

---

Except when responding to a HEAD request, the server SHOULD send a representation containing an explanation of the error situation, and whether it is a temporary or permanent condition. A user agent SHOULD display any included representation to the user.